bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-assessment",
   "version": "2.1.1",
-  "framework": "^2.1.3",
+  "framework": "~2.2.0",
```

Comment:
Could this be changed to `>=2.2` please so it satisfies future releases of the framework.